### PR TITLE
LPS-78491 Create ContextProviderHelper in ddm-api to allow APIO consume it.

### PR DESCRIPTION
**Pull request by victorlaerte**

### CODE REVIEW

**Comment by victorlaerte** [comment]
ci:test

**Comment by victorlaerte** [comment]
ci:test

**Comment by victorlaerte** [comment]
ci:test

**Comment by victorlaerte** [comment]
ci:test

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java [hunk 1]
```diff
@@ -50,6 +50,11 @@ public int countByKeywords(long companyId, long groupId, String keywords) {
 		return doCountByKeywords(companyId, groupId, keywords, false);
 	}
 
+	@Override
+	public int countByC_G(long companyId, long groupId) {
+		return countByKeywords(companyId, groupId, null);
```
Comment (rafaprax) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java]: Hey @victorlaerte,
 In this case, you should use the `filterCountByKeywords` method instead and rename the `countByC_G ` method to `filterCountByC_G` too.

 Can you please fix it and resend? Thx :)

**Comment by victorlaerte** [comment]
@rafaprax done

**Comment by rafaprax** [comment]
Hey @victorlaerte, You forgot to add the LPS at this commit message https://github.com/rafaprax/liferay-portal/pull/425/commits/62544fb516d8104c4ffd6a96d7bec683a193aa81. Please fix it. Thx

**Comment by rafaprax** [comment]
LGTM. @pedroqueiroz Could you please double check the @victorlaerte 's changes? Thx

**Comment by pedroqueiroz** [comment]
LGTM! @rafaprax @victorlaerte

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/56627. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**